### ISSUE TITLE
Add localization doc to https://www.mapbox.com/android-docs/navigation

### ISSUE THREAD

**Issue by samfader** [mapbox/android-docs#759]
It can sometimes be a bit unclear what languages are supported in both text and speech form in the Android Navigation SDK. There's a super excellent document in the GitHub repo that details language support here: https://github.com/mapbox/mapbox-navigation-android/blob/master/localization.md

It seems like it would be helpful to the end-user if this doc were added somewhere in our https://www.mapbox.com/android-docs/navigation content, rather than just buried in this repo. I am happy to work on doing this, but I'd love to hear from the @mapbox/docs on where and how to structure this within the existing content.

cc @1ec5 @riastrad @langsmith

**Comment by HeyStenson** [comment]
"Localization" would be a good topic for a top-level guide in https://www.mapbox.com/android-docs/navigation/overview/. My only worry is that the sidebar list of guides is getting a little long - and, as it is, doesn't really have logical flow of topics. 

I'm not sure what the solution for this is, though. We might need to reconsider how we visually present items in the sidebar so that we could allow for grouping connected concepts. (For example, "Navigation SDK offline support" could be a sidebar section heading, which could contain the "Off-Route Detection" and "Offline Routing" guides.) @colleenmcginnis @katydecorah, what do you think? 

**tl;dr**
@samfader I think this content would be a great addition as a guide in https://www.mapbox.com/android-docs/navigation/overview/. It'd be awesome if you have time to work on a PR! Otherwise I can do it. CAVEAT: The list of guides is getting out of control and we (@mapbox/docs) may need to look at how we handle grouping and displaying them in the sidebar. This won't block work on adding the localization info, but it's something we need to start thinking about.

**Comment by langsmith** [comment]
cc @mapbox/navigation-android that this conversation is happening about navigation docs

**Comment by danesfeder** [comment]
I think this is a great idea @samfader thanks for pointing that out!  I also agree with @HeyStenson, but I'll leave the structuring of the docs to ya'll - you've already made a ton of improvements with the flow of the docs (which we are very grateful for)
